### PR TITLE
Initial manual validation plan

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by rchande** [comment]
cc @DustinCampbell @TheRealPiotrP @akshita31

File: test-plan.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+#### Opening projects
+When you open a directory in VS Code, the C# extension should look for a .csproj or .sln file in that directory and use "OmniSharp" to load it. 
+If you look in "Output > Omnisharp Log" a bunch of information should be printed about what copy of MSBuild was used and what projects were load
+
+Project types to test:
+* Standalone csproj 
+* Directory containing .sln file that references csprojs--projects should be loaded 
+* .NET Core/.NET Standard csproj
+* (Windows) Desktop .NET projects
```
Comment (gregg-miskelly) [inline test-plan.md]: We always get a bunch of feedback on Unity projects. Should we add something there too?

Comment (DustinCampbell) [reply test-plan.md]: Yes. I've typically validated Unity projects.

Comment (rchande) [reply test-plan.md]: Sure--do you have any examples of unity projects people edit?

Comment (DustinCampbell) [reply test-plan.md]: I sent some pointers in email.

**Comment by rchande** [comment]
@DustinCampbell Done--I made the section about unit tests more obvious.

File: test-plan.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# Validating C# Extension for VS Code
+
+#### Opening projects
+When you open a directory in VS Code, the C# extension should look for a .csproj or .sln file in that directory and use "OmniSharp" to load it. 
+If you look in "Output > Omnisharp Log" a bunch of information should be printed about what copy of MSBuild was used and what projects were load
+
+Project types to test:
+* Standalone csproj
```
Comment (TheRealPiotrP) [inline test-plan.md]: Where do we expect these projects to come from?

File: test-plan.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+* (this is not an exhaustive list)
+
+#### Code Lens - References
+* References codelens appears on symbols and shows correct Find All References results
+
+#### Code Lens - Unit Tests
+* In unit tests projects, the "run test" and "debug test" codelens appears on test methods
+  * Clicking runs or debugs the test and prints results to the console
+  * Breakpoints in a unit test are hit if you "debug test"
+
```
Comment (akshita31) [inline test-plan.md]: @rchande Should we mention about "run all tests" and "debug all tests" codelens that was recently added (https://github.com/OmniSharp/omnisharp-vscode/pull/1961) ?

File: test-plan.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+* Try editing a project like https://github.com/staceyhaffner/SuperSpaceShooter
+
+#### Intellisense 
+* The completion list in a file contains symbols defined in references and in the file
+* If you write a documentation comment on a symbol, the completion list displays it
+
+#### Signature Help 
+* Signature Help shows up in a method call after typing `(`
+* Signature help shows documentation for methods and for parameters
+
```
Comment (akshita31) [inline test-plan.md]: Maybe we can add something related to the active parameter here, like "Signature Help shows the documentation for the parameter that is currently active"

Comment (rchande) [reply test-plan.md]: Done

File: test-plan.md (hunk 3 above)
Comment (rchande) [reply test-plan.md]: Done

File: test-plan.md (hunk 2 above)
Comment (rchande) [reply test-plan.md]: I took a workitem for myself to create this corpus: https://github.com/OmniSharp/omnisharp-vscode/issues/2079

**Pull request merged**